src/containerapp/azext_containerapp/_utils.py

```diff
@@ -436,10 +437,11 @@ def _get_oryx_run_image_version_key(tag_obj, framework):
 def parse_oryx_run_image(image_repository, tag) -> OryxRunImageTagProperty:
     # Example Oryx run image: mcr.microsoft.com/oryx/<platform>:<version>-<os>
     # Note: image_repository should NOT include the "mcr.microsoft.com/" prefix; it should ONLY be the repository
-    re_matches = re.findall(r"oryx\/([A-Za-z]*):([0-9.]*)-([A-Za-z-]*)", "{}:{}".format(image_repository, tag))
-    if len(re_matches) == 0:
+    re_matches = re.findall(r"(oryx|azure-buildpacks)\/([A-Za-z]*):([0-9.]*)-([A-Za-z-]*)",
+                            "{}:{}".format(image_repository, tag))
+    if len(re_matches) == 0 or len(re_matches[0]) < 4:
         return None
```

Comment:
does it have to be limited to 4?

Reply:
There's 4 groups in regex, less than 4 means it's not matched.

Reply:
For regular expression-related code, we always recommend adding some examples or descriptions, otherwise no one except the author will be able to understand the meaning of these codes.